src/hello-world/hello-world.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {DebugElement} from '@angular/core';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing';
+
+import {HelloWorldComponent} from './hello-world.component';
+import {HelloWorldModuleNgSummary} from './hello-world.module.ngsummary';
+
+// TODO(alexeagle): this helper should be in @angular/platform-browser-dynamic/testing
+try {
+  TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+} catch {
+  // Ignore exceptions when calling it multiple times.
+}
+
+describe('BannerComponent (inline template)', () => {
+
+    let comp:    HelloWorldComponent;
+    let fixture: ComponentFixture<HelloWorldComponent>;
+    let de:      DebugElement;
```

Comment:
nit: I'm not sure its worth keeping this unused variable around. Up to you if you want to establish this as a common pattern

Reply:
removed